src/modules/listItem.js

```diff
@@ -1,29 +1,63 @@
-export default class ListItem {
+export default class Item {
   constructor(description, completed = false, index = new Date().getTime()) {
```

Comment:
- Please, kindly ensure the property index is set to the value of the array length as this is a major requirement for this project to be approved.